content/sensu-core/0.29/faq.md

```diff
@@ -136,4 +136,21 @@ $ ps aux | grep [s]ensu-server
 sensu     5992  1.7  0.3 177232 24352 ...
 $ kill -TRAP 5992{{< /highlight >}}
 
-[downloads]: https://sensuapp.org/downloads
\ No newline at end of file
+> How can I print my Sensu configuration for troubleshooting?
+
+Frequently, Sensu staff or community members may ask you to print your configuration. It's fairly easy to print the configuration for your Sensu deployment:
+
+**Sensu Core**:
+`/opt/sensu/bin/sensu-client --print_config | tee sensu-core-config.json`
+
+**Sensu Enterprise**
+`sudo -u sensu java -jar /usr/lib/sensu-enterprise/sensu-enterprise.jar -c /etc/sensu/config.json -d /etc/sensu/conf.d --print_config | tee se-config.json`
+
+> RabbitMQ is giving me an error about `wrong credentials`, but everything seems correct. What do I do?
+
+Due to [AMQP's][1] implementation in Sensu, it's often difficult to distinguish between a SSL handshake failure from a bad username/password combination. If you've ensured that the username/password combination in your configuration is correct, we encourage you to check your RabbitMQ/Erlang versions against [RabbitMQ's "Which Erlang" article][2] to see if your versions are able to reliably support TLS. It's also worth noting that as of Sensu 0.27, our build processes changed and we [upgraded the version of OpenSSL][3], and upgrading your client (if < 0.27) may solve the issue.

```

Comment:
"it's often difficult to distinguish between a SSL handshake failure from a bad username/password combination."

Consider striking "between" so it reads "it's often difficult to distinguish a SSL handshake failure from a bad username/password combination."